components/table/table-wrapper.js

```diff
@@ -368,6 +437,7 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 			r.classList.toggle('d2l-table-selected-first', firstNonHeaderRow && isSelected);
 
 			Array.from(r.cells).forEach((c, index) => {
+				if (isHeader && !CSS.supports('selector(:has(a, b))')) this._checkSiblingSortableCells(c);
```

Comment:
With `:has` we are able to do everything without js, so that's why there is a separate `:has` section in the css. This can be removed once we only support browsers with `:has`